05_syntax_lab/01.py

```diff
@@ -1,5 +1,29 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+#Week01_Exercise01_Assignment01
+nIterations = 10
+MaxVal = float("-inf")

```

Comment:
Python variable names are written in lowercase, so prefer `maxval`. Long name with multiple words use underscore notation such as `maximum_value_from_user`